packages/astro/src/core/config/schema.ts

```diff
@@ -43,8 +43,6 @@ const ASTRO_CONFIG_DEFAULTS: AstroUserConfig & any = {
 	experimental: {
 		assets: false,
 		hybridOutput: false,
-		customClientDirectives: false,
-		middleware: false,
```

Comment:
Looks like this wasn't removed in the unflag middleware PR, which I removed here.